mod.conf

```diff
@@ -1,2 +1,3 @@
 name = pick_axe_tweaks
-description = adds the ability for pick axes to place light nodes on right click
+description = Adds the ability for pickaxes to place light nodes. Configurable per player.
+min_minetest_version = 5.1
```

Comment:
Am I forgetting a reason this doesnt work on 5.0?

Reply:
It does work with 5.0, but the `pickaxe` group was added to the default pickaxes in 5.1, so I thought it would be better to set it to that.

Reply:
could polyfill support for it, or just keep it compat for supporting other mods, but 5.0 old enough its not worth the effort